### ISSUE TITLE
Creating a loop (not submitting workflow) causes an uninterruptible error.

### ISSUE THREAD

**Issue by GalaxyTimeMachine** [comfyanonymous/ComfyUI#2077]
I accidentally created a loop by taking the text output of a node and feeding it back into the input of the same node. This creates a pop-up error with an "OK" button, but clicking on the button just cycles the error back again. There is no way to rectify the error because the UI is unresponsive while the error show.
![image](https://github.com/comfyanonymous/ComfyUI/assets/52193044/32e05e01-eb61-4b69-9746-fbeda0fa587f)

**Comment by SmokeyRGB** [comment]
Seconding, really annoying if you've worked on a workflow for some time and are messing up all your work by making a small mistake.

Only way to get rid of the alert box is to close the window, loosing all progress made on the workflow.

**Comment by GalaxyTimeMachine** [comment]
Even closing the window doesn't work because the same workflow reloads when you open it again. The only way I could clear it was to clear the browser cache, and so I lost all of my node templates as well :(

**Comment by Smoky78** [comment]
Are you using UE Nodes - [cg-use-everywhere](https://github.com/chrisgoringe/cg-use-everywhere) ? If so, this caused this exact error with me.

**Comment by GalaxyTimeMachine** [comment]
Yes, I am using the "everywhere" nodes. Perhaps this bug needs to move over to there.

**Comment by SmokeyRGB** [comment]
Same, seems to be a problem with the UE-Nodes then.
I was wondering why they would randomly introduce such a weird way of error-avoidance at this point. 
Thanks @Smoky78

**Comment by chrisgoringe** [comment]
Thanks for raising the bug across at UE - fixed in the latest update.

**Comment by mr-bob-chang** [comment]
I didn't use the cg-use-everywhere node, but I still run into this problem, This problem occurs when connecting ApplyControlnet for Inpaint to FaceDetailer node.
![企业微信截图_20240408112150](https://github.com/comfyanonymous/ComfyUI/assets/141008527/3c606838-845e-4b66-830d-97d515c9b346)

**Comment by chrisgoringe** [comment]
@mr-bob-chang posting the workflow, or at least a screenshot, would be helpful.

note that you can turn the loop checking off in the main settings:
![Screenshot 2024-04-08 134725](https://github.com/comfyanonymous/ComfyUI/assets/4371971/76e8c893-3add-428e-b25b-9dae432e2e41)

**Comment by YellowjacketXVI** [comment]
> I accidentally created a loop by taking the text output of a node and feeding it back into the input of the same node. This creates a pop-up error with an "OK" button, but clicking on the button just cycles the error back again. There is no way to rectify the error because the UI is unresponsive while the error show. ![image](https://private-user-images.githubusercontent.com/52193044/285907006-32e05e01-eb61-4b69-9746-fbeda0fa587f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzMTM0MDEsIm5iZiI6MTcxODMxMzEwMSwicGF0aCI6Ii81MjE5MzA0NC8yODU5MDcwMDYtMzJlMDVlMDEtZWI2MS00YjY5LTk3NDYtZmJlZGEwZmE1ODdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDIxMTE0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlOTk1YzA1ZjQ1ZTk4YjhiOTQxNjVlNTNhODkxZDAxNmZkYWY2NmYyODE0MDU3NjkyYmZiNTAwMDUxOWRkZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9BjdJZbFLNhPqkAXdDv5oHYF3kpPSrJ3ihcn9Xp2VkE)

You a node creating a loop as in something that should be another node is actually happening after and creating a loop, just double check all you nodes, make sure the latent is linear in one direction

**Comment by LPG-VERSION** [comment]
I remember that the connection loop would report an error and the recursion depth would be too large

**Comment by ani917** [comment]
![wt](https://github.com/user-attachments/assets/535b70ca-aea4-4138-af17-df5dfc520e34)
this circuit causes an error

**Comment by chrisgoringe** [comment]
If it is the UE nodes, you can turn off loop checking in the settings
